### PR TITLE
chore(generator): IWorker

### LINKED ISSUE #1252: Use generated IWorker

### PR DESCRIPTION
**Pull request by avodovnik**
Closes #1252

### CODE REVIEW

**Comment by avodovnik** [comment]
We can't merge this, unless https://github.com/microsoft/playwright/pull/5730 merges, because we're expecting the method to become `async`.

File: src/PlaywrightSharp/Contracts/IWorker.generated.cs
Comment (kblok) [inline src/PlaywrightSharp/Contracts/IWorker.generated.cs:123]: Is this a bug in the generator?

File: src/PlaywrightSharp/Contracts/Models/UndefinedEvaluationArgument.cs
Comment (kblok) [inline src/PlaywrightSharp/Contracts/Models/UndefinedEvaluationArgument.cs:17]:
```suggestion
```

File: src/PlaywrightSharp/Worker.cs
Comment (kblok) [inline src/PlaywrightSharp/Worker.cs:76]: Why do we need this? Can't we use `EvaluateArgument.Undefined` and that's it?

File: src/PlaywrightSharp/Contracts/IWorker.generated.cs
Comment (avodovnik) [reply src/PlaywrightSharp/Contracts/IWorker.generated.cs:123]: No, it's due to the fact we're migrating one by one. The reference *will* be there, when we move all the interfaces over.

File: src/PlaywrightSharp/Worker.cs
Comment (avodovnik) [reply src/PlaywrightSharp/Worker.cs:76]: We _could_, but it would mean making a whole bunch of the other stuff public, that's currently `internal` and I would much much rather have a separate object. It also doesn't mess with the logic of where it's used (i.e. it's not used for transport, but it gets translated) and that will bring it inline to where we want to be, mid-term future. Hence my decision to go with a separate one.

**Review by kblok — APPROVED** [review]

**Pull request merged**
